spec/abstractops.html

```diff
@@ -29,6 +29,116 @@ <h1>IterableToListOfType ( _items_, _elementTypes_ )</h1>
+          _date_: an Integer,
+        ): an Integer
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It calculates a number of days.</dd>
+      </dl>
+      <emu-alg>
+        1. Let _ym_ be _year_ + (floor(_month_) / 12).
+        1. Let _mn_ be _month_ modulo 12.
+        1. Find a time value _t_ such that YearFromTime(_t_) is _ym_, MonthFromTime(_t_) is _mn_, and DateFromTime(_t_) is 1.
+        1. Return ℝ(Day(_t_)) + _date_ - 1.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-makedate" type="abstract operation">
+      <h1>
+        MakeDate (
+          _day_: a Number,
+          _time_: a Number,
+        ): a Number
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It calculates a number of milliseconds.</dd>
+      </dl>
+      <emu-alg>
+        1. Return be _day_ × ℝ(msPerDay) + _time_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-year-number">
+      <h1>Year Number</h1>
+      <p>ECMAScript uses a proleptic Gregorian calendar to map a day number to a year number and to determine the month and date within that year. In this calendar, leap years are precisely those which are (divisible by 4) and ((not divisible by 100) or (divisible by 400)). The number of days in year number _y_ is therefore defined by</p>
+      <emu-eqn id="eqn-DaysInYear" aoid="DaysInYear">
+        DaysInYear(_y_)
+          = 365 if ((_y_) modulo 4) ≠ 0
+          = 366 if ((_y_) modulo 4) = 0 and ((_y_) modulo 100) ≠ 0
+          = 365 if ((_y_) modulo 100) = 0 and ((_y_) modulo 400) ≠ 0
+          = 366 if ((_y_) modulo 400) = 0
+      </emu-eqn>
+      <p>All non-leap years have 365 days with the usual number of days per month and leap years have an extra day in February. The day number of the first day of year _y_ is given by:</p>
+      <emu-eqn id="eqn-DaysFromYear" aoid="DayFromYear">DayFromYear(_y_) = 365 × (_y_ - 1970) + floor((_y_ - 1969) / 4) - floor((_y_ - 1901) / 100) + floor((_y_ - 1601) / 400)</emu-eqn>
+      <p>The time value of the start of a year is:</p>
+      <emu-eqn id="eqn-TimeFromYear" aoid="TimeFromYear">TimeFromYear(_y_) = ℝ(msPerDay) × DayFromYear(_y_)</emu-eqn>
+      <p>A time value determines a year by:</p>
+      <emu-eqn id="eqn-YearFromTime" aoid="YearFromTime">YearFromTime(_t_) = the largest integral Number _y_ (closest to +∞) such that TimeFromYear(_y_) ≤ _t_</emu-eqn>
+      <p>The leap-year function is 1 for a time within a leap year and otherwise is 0:</p>
+      <emu-eqn id="eqn-InLeapYear" aoid="InLeapYear">
+        InLeapYear(_t_)
+          = 0 if DaysInYear(YearFromTime(_t_)) is 365
+          = 1 if DaysInYear(YearFromTime(_t_)) is 366
+      </emu-eqn>
+    </emu-clause>
+
+    <emu-clause id="sec-month-number">
+      <h1>Month Number</h1>
+      <p>Months are identified by an integer in the inclusive interval from 0 to 11. The mapping MonthFromTime(_t_) from a time value _t_ to a month number is defined by:</p>
```

Comment:
I would work on the description for this a bit, in particular maybe a note why this is using 0-based months when we use 1-based months in the rest of Temporal.